inlong-manager/manager-schedule/src/main/java/org/apache/inlong/manager/schedule/airflow/config/AirflowConfig.java

```diff
@@ -38,7 +38,7 @@
 @EqualsAndHashCode(callSuper = true)
 public class AirflowConfig extends ClientConfiguration {
 
-    @Value("${schedule.engine.inlong.manager.host:127.0.0.1}")
+    @Value("${schedule.engine.inlong.manager.url:127.0.0.1}")
```

Comment:
The URL should include port information. Does this parameter meet the definition?

Reply:
done.